pipeline/terraform/stack/tei_off_branch.tf

```diff
@@ -0,0 +1,55 @@
+module "tei_off_branch" {
+  source = "./branch"
```

Comment:
Is it worth naming this branch? Just in case we want to keep it later, it'll will be a bit harder to rename later (althought we've already applied this it seems).

Reply:
yeah, I don't love the name, but I'm not sure what a good name would be. It os true that is already applied but it's not a particularly massive effort to rename this

Reply:
No need, we can do it if we find it useful when spinning up new pipelines.